src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  onAddTodo: (title: string, userId: number) => void;
+}
+
+export const TodoForm: React.FC<Props> = ({ onAddTodo }) => {
```

Comment:
Destructure the `users` prop along with `onAddTodo` to use the passed user data instead of importing it directly.